src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -876,6 +876,8 @@ def call(self, input_ids, use_cache=False):
 )
 @keras_serializable
 class TFBartModel(TFPretrainedBartModel):
+    base_model_prefix = "model"
```

Comment:
Was this previously missing? It is in the `TFPretrainedBartModel`:

```py
class TFPretrainedBartModel(TFPreTrainedModel):
    config_class = BartConfig
    base_model_prefix = "model"
```

Reply:
Yes, it was missing.